### DISCUSSION TITLE
Making the btc_p_mil_group_ratio even higher?

### DISCUSSION THREAD

**Discussion by NoAce4U** [Vdauphin/HeartsAndMinds#1157] [Question and answer]
I see in the mission.sqf there is 

`btc_p_mil_group_ratio = ("btc_p_mil_group_ratio" call BIS_fnc_getParamValue)/100;`

From my understanding this is taking the value, and dividing by 100?

Would lowering the 100 increase the enemy count or decrease? 

Or is there somewhere else I should be looking to increase the enemy count

**Comment by Vdauphin** [comment]
Hello,
> From my understanding this is taking the value, and dividing by 100?

It is just a way to go from a pourcentage to the group ratio value

> Would lowering the 100 increase the enemy count or decrease?

Just add 110% or 150% here:
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L124-L126

> Or is there somewhere else I should be looking to increase the enemy count

The actual implementation make number of group spawning in a city random: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/activate.sqf#L122
I think instead of increase the group ratio you should just remove the `random` so you are sure to have the maximum of group for the type of city activating
Also note that your change will not affect city previously activated because it is persistent 
Cheers

**Reply by NoAce4U** [reply]
Okay, my next question is....

I'm seeing a lot of groups of 1 or 2, Is there a way to change the value of group sizes?

**Reply by Vdauphin** [reply]
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/activate.sqf#L123

`1 + round random [0, 1, 2]` just put `3` or whatever

**Comment by Vdauphin** [comment]
This behavior will change since 1.22 (https://github.com/Vdauphin/HeartsAndMinds/pull/1111)
